packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -48,5 +48,8 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     },
   });
   const state = decodeOAuthState(req);
-  res.redirect(getSafeRedirectUrl(state?.returnTo) ?? "/apps/installed");
+  res.redirect(
+    getSafeRedirectUrl(state?.returnTo) ??
+      getInstalledAppPath({ variant: "calendar", slug: "google-calendar" })
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.